query-engine/connectors/sql-query-connector/src/sql_info.rs

```diff
@@ -11,6 +11,16 @@ use std::env;
 pub static BATCH_SIZE_OVERRIDE: Lazy<Option<usize>> =
     Lazy::new(|| env::var("QUERY_BATCH_SIZE").ok().and_then(|size| size.parse().ok()));
 
+#[cfg(not(test))]
+fn get_batch_size(default: usize) -> Option<usize> {
+    (*BATCH_SIZE_OVERRIDE).or(Some(default))
+}
+
+#[cfg(test)]
+fn get_batch_size(_: usize) -> Option<usize> {
+    env::var("QUERY_BATCH_SIZE").ok().and_then(|size| size.parse().ok())
+}
+
```

Comment:
When testing, we should always read from the environment.
This allows individual tests to override the `QUERY_BATCH_SIZE` variable if needed (spoiler: I needed it in this PR 😄)

Reply:
As I have commented the only test that needed to tinker with `QUERY_BATCH_SIZE`, this change to `sql_info.rs` is technically no longer needed. Should I get rid of it?